campfire/src/golden_images/mod.rs

```diff
@@ -0,0 +1,251 @@
+// Intentionally overrides the global Clippy settings because Campfire is not
+// part of Ambient engine.
+#![allow(clippy::disallowed_types)]
+
+use anyhow::{bail, Context};
+use clap::Parser;
+use itertools::Itertools;
+use std::{process::Command, time::Instant};
+
+mod failure;
+use failure::*;
+
+mod progress;
+use progress::*;
+
+const TEST_BASE_PATH: &str = "guest/rust/examples";
+const TESTS: &[&str] = &[
```

Comment:
Is there a way to avoid hardcoding this? Maybe have a file at the root of `guest/rust/examples` that lists the projects to test?

Reply:
I can add a manifest under `guest/rust/examples` which lists all tests.